tests/pages/api/grafanaAPI.js

```diff
@@ -125,4 +127,50 @@ module.exports = {
       `Failed to set custom Home dashboard '${id}'. Response message is ${resp.data.message}`,
     );
   },
+
+  async createFolder(name) {
+    const headers = { Authorization: `Basic ${await I.getAuth()}` };
+    const body = {
+      title: name,
+    };
+    const resp = await I.sendPostRequest('graph/api/folders', body, headers);
+
+    assert.ok(
+      resp.status === 200,
+      `Failed to create "${name}" folder. Response message is ${resp.data.message}`,
+    );
+
+    return resp.data;
+  },
+
+  async lookupFolderByName(name) {
+    const headers = { Authorization: `Basic ${await I.getAuth()}` };
+    const resp = await I.sendGetRequest('graph/api/folders', headers);
+
+    return Object.entries(resp.data).filter((folder) => folder.title === name);
+  },
+
+  async deleteFolder(uid) {
+    const headers = { Authorization: `Basic ${await I.getAuth()}` };
+
+    const resp = await I.sendDeleteRequest(`graph/api/folders/${uid}`, headers);
+
+    assert.ok(
+      resp.status === 200,
+      `Failed to delete folder with uid '${uid}' . Response message is ${resp.data.message}`,
+    );
+  },
+
+  /**
+   * Encapsulates multiple actions require do delete folder to keep test body short.
+   */
+  async safeDeletePmmFolder() {
+    const searchResultObj = await this.lookupFolderByName('PMM');
+
+    if (searchResultObj && Object.prototype.hasOwnProperty.call(searchResultObj, 'uid')) {
+      await this.deleteFolder(searchResultObj.uid);
```

Comment:
I mentioned that PMM folder was not always empty, so why deleting it?
![image](https://user-images.githubusercontent.com/3048593/144544592-55116872-52cc-45e9-87ba-4a03abb649d6.png)

Reply:
removed tis strategy and added if instead.